recipe/meta.yaml

```diff
@@ -32,11 +32,11 @@ test:
         - test -f $PREFIX/lib/libflann_s.a                                        # [unix]
         - test -f $PREFIX/lib/libflann.dylib                                      # [osx]
         - test -f $PREFIX/lib/libflann.so                                         # [linux]
-        - test -f $PREFIX/lib/libflann.dll                                        # [win]
         - test -f $PREFIX/lib/libflann_cpp_s.a                                    # [unix]
         - test -f $PREFIX/lib/libflann_cpp.dylib                                  # [osx]
         - test -f $PREFIX/lib/libflann_cpp.so                                     # [linux]
-        - test -f $PREFIX/lib/libflann_cpp.dll                                    # [osx]
+        - if not exist %PREFIX%\\Library\\lib\\libflann.lib exit 1                # [win]
+        - if not exist %PREFIX%\\Library\\bin\\libflann.dll exit 1                # [win]
```

Comment:
I am guessing we will have the `_cpp` ones too. Also, I don't know if they still do this `_s` thing on static libraries with Windows too (which is a kind of neat solution, sad that compilers still don't have a better way to solve this).